code/game/objects/structures/signs/signs_departments.dm

```diff
@@ -1,96 +1,164 @@
 //departmental signs
+//sign_change_name is an alternate name for a sign that only shows up in the list you see when you use a pen on a sign backing.
+
+
+///////MEDBAY
+
+/obj/structure/sign/departments/medbay
+	name = "\improper MEDBAY"
+	sign_change_name = "Department - Medbay"
+	desc = "The Intergalactic symbol of Medical institutions. You'll probably get help here."
+	icon_state = "bluecross"
+	is_editable = TRUE
+
+/obj/structure/sign/departments/medbay/alt
+	name = "\improper MEDBAY"
+	sign_change_name = "Department - Medbay Alt"
+	desc = "The Intergalactic symbol of Medical institutions. You'll probably get help here."
+	icon_state = "bluecross2"
+	is_editable = TRUE
```

Comment:
make a base department sign/sign that has is_editable as TRUE as you seem to be setting it on every subtype of departments

Reply:
As per our discussion elsewhere, there doesn't seem to be a better way to do it offhand.